src/locales/de/ability-trigger.ts

```diff
@@ -10,6 +10,7 @@ export const abilityTriggers: SimpleTranslationEntries = {
   "trace": "{{pokemonName}} kopiert {{abilityName}} von {{targetName}}!",
   "windPowerCharged": "Der Treffer durch {{moveName}} läd die Stärke von {{pokemonName}} auf!",
   "quickDraw": "Durch Schnellschuss kann {{pokemonName}} schneller handeln als sonst!",
+  "disguiseAvoidedDamage" : "{{pokemonName}}'s disguise was busted!",
```

Comment:
```suggestion
  "disguiseAvoidedDamage" : "Die Tarnung von {{pokemonName}} is aufgeflogen!",
```
[source 15:22](https://youtu.be/xlt7QtypSqI?si=ZVRASsn9sKuIVQLQ)

Reply:
```
Die Tarnung von {{pokemonName}} ist aufgeflogen!
```

Reply:
(you are missing a t in ist)